sml_builder/method.py

```diff
@@ -11,6 +11,17 @@ def display_method_summary(method, methodState):
     page_data = loads(
         evaluate_file(f"./content/methods/{methodState}/{method}.jsonnet")
     )
+    sorted_order = [
```

Comment:
This would be a good place to add a short comment to explain why we need to re-order, it may help someone else who faces a similar issue later on.